src/libs/ReportUtils.ts

```diff
@@ -3464,14 +3477,63 @@ function shouldHideReport(report: OnyxEntry<Report>, currentReportId: string): b
     return parentReport?.reportID !== report?.reportID && !isChildReportHasComment;
 }
 
+function transactionHasViolation(transactionID: string, transactionViolations: TransactionViolations): boolean {
```

Comment:
I prefer `hasTransactionViolation` to be consistent with other boolean function names

Reply:
I like it as is, but if we're going to update it, I suggest `hasViolation`. `ReportUtils.hasTransactionViolation` sounds like we'll do the checks on the report and not the violations.

We could also move this to `TransactionUtils.hasViolations`, but this isn't a blocker for me

Reply:
Moved to `TransactionUtils.hasViolations`